### DISCUSSION TITLE
Is there a list of all possible achivements?

### DISCUSSION THREAD

**Discussion by devkennyy** [lowlighter/metrics#1161] [Q&A] [answered]
Within the achievements plugin, I'm wondering if it were possible to see all possible achievements?

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

You can see the whole list and threshold for each rank in [here](https://github.com/lowlighter/metrics/blob/master/source/plugins/achievements/list/users.mjs):
https://github.com/lowlighter/metrics/blob/6699d7ecc26d8a60170a5ae74bb2367518340efe/source/plugins/achievements/list/users.mjs#L9-L397

Alternatively you can set [`plugin_achievements_threshold: X`](https://github.com/lowlighter/metrics/tree/master/source/plugins/achievements#plugin_achievements_threshold) to display all achievements, including the one not unlocked

This contains all achievement except one IIRC (the one requiring to create a user project): 
<img src="https://gist.githubusercontent.com/lowlighter/3c6eaedf50273adfb7a510822672f570/raw/achievements.svg">

Hope this helps 👍
